src/main/java/org/prebid/server/auction/BidResponsePostProcessor.java

```diff
@@ -17,12 +17,12 @@
      * This method is called when auction is finished.
      *
      * @param context     represents initial web request
-     * @param bidRequest  original auction request
      * @param uidsCookie  auction request {@link Uids} container
+     * @param bidRequest  original auction request
      * @param bidResponse auction result
      * @return a {@link Future} with (possibly modified) auction result
      */
-    Future<BidResponse> postProcess(RoutingContext context, BidRequest bidRequest, UidsCookie uidsCookie,
```

Comment:
This is seemingly a breaking change. I know @jordillachmrf may have used this to cut in their specific logic in auction flow so it may a good idea to reach him before going forward with this change

Reply:
The change is fine, thanks for letting me know